src/wasm.cc

```diff
@@ -394,7 +394,10 @@ void WasmBase::startVm(ContextBase *root_context) {
              // time
              "wasi_unstable.clock_time_get", "wasi_snapshot_preview1.clock_time_get",
              // random
-             "wasi_unstable.random_get", "wasi_snapshot_preview1.random_get"});
+             "wasi_unstable.random_get", "wasi_snapshot_preview1.random_get",
+             // For proxy-wasm-go-sdk Go runtime initialization
```

Comment:
Nit: `Go runtime initialization` would be sufficient, since technically it's not specific to proxy-wasm-go-sdk SDK.

Reply:
Done.